src/tests/control/test_items.py

```diff
@@ -445,3 +445,36 @@ def test_create_copy(self):
         assert i_new.allow_cancel == i_old.allow_cancel
         assert set(i_new.questions.all()) == set(i_old.questions.all())
         assert set([str(v.value) for v in i_new.variations.all()]) == set([str(v.value) for v in i_old.variations.all()])
+
+    def test_add_to_existing_quota(self):
+        q = Quota.objects.create(event=self.event1, name="New Test Quota", size=50)
+
+        doc = self.get_doc('/control/event/%s/%s/items/add' % (self.orga1.slug, self.event1.slug))
+        form_data = extract_form_fields(doc.select('.container-fluid form')[0])
+        form_data['name_0'] = 'Existing'
+        form_data['default_price'] = '2.00'
+        form_data['quota_option'] = 'existing'
+        form_data['quota_add_existing'] = '1'
```

Comment:
You should use ``str(q.pk)`` here instead ``"1"``. It's not guaranteed that this ID will be always 1, which is why the tests currently fail on PostgreSQL and MySQL.